index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Kindly edit the pull request description. Note that your pull request description should be a brief summary of all changes you implemented in the branch.
![image](https://user-images.githubusercontent.com/94551484/219792803-e740a57a-4dd7-46fc-af81-dc2b3b6e5bfb.png)